### DISCUSSION TITLE
parametro attachment_url in List Received Documents

### DISCUSSION THREAD

**Discussion by zpj61** [fattureincloud/api#231] [Q&A] [answered]
Nella documentazione risulta che con il parametro "attachment_url" si ottiene un PDF ma in realtà è sempre un XML. Resterà sempre così? Cosa dovrebbe ritornare "attachment_preview_url" che nelle nostre prove è sempre null?

**Comment by emajo** [comment, marked as answer]
Ciao @zpj61 campo `attachment_url` contiene il file che è stato caricato come allegato in creazione del documento, perciò se viene caricato un pdf conterrà il pdf, in caso di documento elettronico probabilmente sarà sempre un xml, `attachment_preview_url` invece contiene un jpg contenente la preview del documento in caso questo sia un pdf.

Grazie,
Emanuele.
